### PR TITLE
Move deployement back to this repo, and add voting CI with a localhost deploy

### PR DESCRIPTION
**Pull request by mikalstill**
I may regret this.

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+Shaken Fist: Opinionated to the point of being impolite
```
Comment (mandoonandy) [inline deploy/README.md:1]: Stray file to see if I'm checking?

File: docs/installation.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ This guide will assume that you want to install Shaken Fist on a single local ma
...
 
-Create a directory for Shaken Fist, and then checkout the deployer git repository there:
+Create a directory for Shaken Fist, and then checkout the Shaken Fist git repository there:
 
 ```bash
 mkdir shakenfist
 cd shakenfist
-git clone https://github.com/shakenfist/deploy
+git clone https://github.com/shakenfist/shakenfist
 cd deploy/ansible
```
Comment (mandoonandy) [inline docs/installation.md:16]: `cd shakenfist/deploy/ansible`

File: docs/installation.md [hunk 3]
```diff
@@ -7,12 +7,12 @@ This guide will assume that you want to install Shaken Fist on a single local ma
 
 Shaken Fist only supports Ubuntu 18.04 or later, so if you're running on localhost that implies that you must be running a recent Ubuntu on your development machine. Note as well that the deployer installs software and changes the configuration of your networking, so be careful when running it on machines you are fond of.
 
-Create a directory for Shaken Fist, and then checkout the deployer git repository there:
+Create a directory for Shaken Fist, and then checkout the Shaken Fist git repository there:
 
 ```bash
 mkdir shakenfist
 cd shakenfist
```
Comment (mandoonandy) [inline docs/installation.md:14]: What do you think about removing these two lines?

I've always thought that anybody using this is smart enough to organise their own directories. Git will create it's own `shakenfist` dir anyway.

File: shakenfist/daemons/main.py [hunk 4]
```diff
@@ -125,7 +125,7 @@ def _start_daemon(d):
             # do is create a bridge if it doesn't exist and the wire everything up
             # to it. We can do egress NAT in that state, even if floating IPs
             # don't work.
-            with util.RecordedOperation('create physical bridge', 'startup'):
+            with util.RecordedOperation('create physical bridge', None):
```
Comment (mandoonandy) [inline shakenfist/daemons/main.py:128]: yay!

**Review by mandoonandy — CHANGES_REQUESTED** [review]

File: deploy/README.md (hunk 1 above)
Comment (mikalstill) [reply deploy/README.md:1]: No, just asleep at the wheel.

Comment (jackadamson) [inline deploy/README.md:1]: Should this be in the docs directory so it is served at http://shakenfist.com/?

File: docs/installation.md (hunk 2 above)
Comment (mikalstill) [reply docs/installation.md:16]: Done.

File: docs/installation.md (hunk 3 above)
Comment (mikalstill) [reply docs/installation.md:14]: Yeah, I agree now that I read it again.

File: shakenfist/daemons/main.py (hunk 4 above)
Comment (mikalstill) [reply shakenfist/daemons/main.py:128]: You're welcome!

File: deploy/README.md (hunk 1 above)
Comment (mikalstill) [reply deploy/README.md:1]: Yes. Especially that installation grid thingie. I'll do that now.

Comment (mikalstill) [reply deploy/README.md:1]: Done.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**